config/initializers/omniauth.rb

```diff
@@ -6,6 +6,7 @@
     prompt: "select_account",
     image_aspect_ratio: "square",
     # we're displaying at 80 pixels, this is for high density ("Retina") displays
-    image_size: 160
+    image_size: 160,
+    hd: ENV["OAUTH2_RESTRICTED_DOMAIN"] || ""
```

Comment:
What about default this to [`ORIENTATION_DOMAIN`](https://github.com/doximity/orientation/blob/master/app.json#L18)? 

Or maybe you want to be able to have a separate Apps domain allowed?